src/CodeReview.Orchestrator/Program.cs

```diff
@@ -50,14 +62,19 @@ private static int ProcessErrors(ParserResult<object> result)
             return Constants.ErrorExitCode;
         }
 
+        private static async Task<int> ProcessEvaluateAsync(NewDockerEngineCollectionManifestOptions options, IServiceProvider container)
+        {
+            return await container.GetRequiredService<ICreateDockerEngineCollectionManifestCommand>().ExecuteAsync(options);
```

Comment:
Please, do not use async/await in such cases. Just return `Task<T>` to avoid using a state machine.